apps/desktop/src/mods/ModBox.tsx

```diff
@@ -138,6 +141,21 @@ export function ModBox(props: IModBoxProps) {
     });
   };
 
+  const updated = (updated: string) => {
+    var diff =
+      (new Date().valueOf() - new Date(updated).valueOf()) / (1000 * 3600 * 24);
+    return diff < 1 ? 'today' : Math.ceil(diff) + ' days ago';
```

Comment:
Not so important but we've got already `dayjs` library, it should be sufficient to add this line to use the relative time utilities. Would be better for years/months + localization
https://day.js.org/docs/en/plugin/relative-time

```tsx
import dayjs from 'dayjs';
import relativeTime from 'dayjs/plugin/relativeTime';
dayjs.extend(relativeTime);
```

(It's not an issue if you'd prefer to skip this one - we could always do it later)